tests/test_helpers/helpers.ts

```diff
@@ -833,6 +833,11 @@ export function getFigureDefinition(
   }
 }
 
+export function getChartConfiguration(model: Model, chartId: UID) {
```

Comment:
duplicated from the one in `chart_helpers.ts`.

And it is worth to type the return value correctly instead of `any`, or does it invovle a lot of ts-ignore beacause chartJS typing sucks ?